packages/atomic/src/components.d.ts

```diff
@@ -2200,6 +2204,10 @@ declare namespace LocalJSX {
           * Defining this option makes the search box standalone.  This option defines the default URL the user should be redirected to, when a query is submitted. If a query pipeline redirect is triggered, it will redirect to that URL instead (see [query pipeline triggers](https://docs.coveo.com/en/1458)).
          */
         "redirectionUrl"?: string;
+        /**
+          * This option controls the timeout for suggestion queries. If a query times out, only the suggestions from that particular query won't be shown.
```

Comment:
@jpmarceau any suggestions on this beautiful sentence?

Reply:
Only minor ones :) 
```suggestion
          * The timeout for suggestion queries, in milliseconds. If a query times out, the suggestions from that particular query won't be shown.
```